### PR TITLE
FIX: IMS class set_current_position limits issue

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

In the `IMS` class and in similar motor classes, `set_current_position` had a bug where it would refuse to let you set a position that is outside of the limits span.
See https://jira.slac.stanford.edu/browse/ECS-5354
This has been fixed by replacing the deprecated `set_and_wait` call (which threw a warning) with a `put(x, force=True)`, which ignores the limit. This is followed by a manual confirmation because `put` doesn't return a status object and the waiting doesn't work anyway for these specific devices (hence the special function override). I would have preferred to do something like `set(x, force=True).wait(timeout=1)` but `set` doesn't currently support the `force` kwarg. I think this is also a bug, but one I'm less motivated to look into.

## Motivation and Context

https://jira.slac.stanford.edu/browse/ECS-5354

## How Has This Been Tested?

Interactively in XPP
I'll think at least for a moment if there's a reasonable unit test to add but there might not be

## Where Has This Been Documented?

Here, jira, + later release notes

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by ZLLentz** [comment]
Writing a unit test for this requires emulating some IOC behavior so it seems somewhat fruitless. I'm going to mark this as ready for review.

File: pcdsdevices/epics_motor.py [hunk 1]
```diff
@@ -601,7 +603,13 @@ def set_current_position(self, pos):
         """
         self.set_use_switch.put(1, wait=True)
         try:
-            set_and_wait(self.user_setpoint, pos, timeout=1)
+            # Force to ignore limits
+            self.user_setpoint.put(pos, force=True)
+            # Instead of waiting on the put (broken), manually check
```
Comment (tangkong) [inline pcdsdevices/epics_motor.py:608]: For my own understanding, what we're doing here is putting the motor into set-mode and setting the user_setpoint to something that could be outside of the limits.  

Does this have the possibility of putting us into a weird state where the setpoint is outside the limits?

Comment (ZLLentz) [reply pcdsdevices/epics_motor.py:608]: When you change the set position using set mode, the limits also get updated appropriately to maintain the motor's apparent distance from them.

Comment (tangkong) [reply pcdsdevices/epics_motor.py:608]: And python doesn't pick up on this until after the set move I guess.  I see

**Review by tangkong — APPROVED** [review]
I agree that the ideal would be waiting on some status instead of just assuming that the process will resolve within 0.3s, but this seems to work and is good enough for now

File: pcdsdevices/epics_motor.py (hunk 1 above)
Comment (ZLLentz) [reply pcdsdevices/epics_motor.py:608]: Yeah, the Python is actually unnecessarily limiting the setpoint update here- it sees that there are limits on the setpoint PV, but the IOC has no intention of enforcing them while in the setter mode.

Comment (ZLLentz) [reply pcdsdevices/epics_motor.py:608]: It's also worth noting also that all other version of this setter code use `force=True`, it must have snuck in as a mistake when re-implementing this here.

**Comment by ZLLentz** [comment]
> I agree that the ideal would be waiting on some status

I'm a bit confused that the built-in waiting mechanisms don't allow me to pass `force` as a kwarg. Might look into this at a later date.

**Comment by ZLLentz** [comment]
Updating branch and then merging if it passes again

**Pull request merged**